backend/danswer/configs/model_configs.py

```diff
@@ -80,11 +80,14 @@
 GEN_AI_LLM_PROVIDER_TYPE = os.environ.get("GEN_AI_LLM_PROVIDER_TYPE") or None
 # Override the auto-detection of LLM max context length
 GEN_AI_MAX_TOKENS = int(os.environ.get("GEN_AI_MAX_TOKENS") or 0) or None
+
 # Set this to be enough for an answer + quotes. Also used for Chat
+# This is the minimum token context we will leave for the LLM to generate an answer
 GEN_AI_MAX_OUTPUT_TOKENS = int(os.environ.get("GEN_AI_MAX_OUTPUT_TOKENS") or 1024)
```

Comment:
This should be renamed to `GEN_AI_NUM_RESERVED_OUTPUT_TOKENS` or something similar?